third_party/btwxt/src/gridpoint.cpp

```diff
@@ -294,15 +294,19 @@ void GridPoint::normalize_grid_values_at_target(const double scalar) {
   set_results();
 }
 
-void GridPoint::normalize_grid_values_at_target(std::size_t table_num, const double scalar) {
+double GridPoint::normalize_grid_values_at_target(std::size_t table_num, const double scalar) {
   if (!target_is_set) {
     showMessage(MsgLevel::MSG_WARN,
                 stringify("Cannot normalize grid values. No target has been set."));
-    return;
+    return scalar;
   }
-  grid_data->normalize_value_table(table_num,results[table_num]*scalar);
+  // create a scalar which represents the product of the inverted normalization factor and the value in the table at the independent variable reference value
```

Comment:
I think the comment here might be confusing you, and rightly so since it starts to become difficult to talk about the scalars inside of btwxt. Lets start by saying that the scalar here exists as it was entered. For example in the `NormalizationAutomaticWithDivisorAndSpecifiedDivisor` unit test, where the normalization divisor is 4, what is returned in `GridPoint::normalize_grid_values_at_target` is 4.

With that being said, btwxt is looking for that scalar to be inverted (e.g. 1.0/scalar). Before the update to btwxt inside this bug fix, we actually relied on the user to input the scalar this way: `btwxtManager.normalizeGridValues(gridIndex, PerfCurve(CurveNum).GridValueIndex, normalizeTarget, 1.0/normalizationDivisor);`
It's much easier on the user to just let them pass in their normalization divisor, so I did away with this. However, now inside of btwxt, what I am calling the "inverted divisor" is really the one that the user entered. Does that make sense? From the btwxt library user's perspective, the scalar is normal here. From btwxt's perspective, it is inverted.

Reply:
*Does that make sense?* To be honest, no, but I'll just accept that it works.